utils/CreateDataset.py

```diff
@@ -105,9 +105,10 @@ def __getitem__(self, index):
             if meta_idx != -1:
                 metadata = self.metadata[meta_idx]
                 metadata = eval(metadata) if isinstance(metadata, str) else metadata  # FIXME!!! must provide dtype
-            sample_indices = hdf5_file["sample_indices"][index, ...]  # Only useful fur debugging
+            sat_img_dtype = hdf5_file["sat_img_dtype"][index, ...][0]  # FIXME: replace
+            # sample_indices = hdf5_file["sample_indices"][index, ...]  # Only useful fur debugging
```

Comment:
Maybe create a `if debugging`, just an idea but that can be useful later I think.